src/Longrunning/OperationsClient.php

```diff
@@ -0,0 +1,478 @@
+        }
+        $this->operationsStub = $this->grpcCredentialsHelper->createStub(
+            $createOperationsStubFunction,
+            $options['serviceAddress'],
+            $options['port'],
+            $createStubOptions
+        );
+    }
+
+    /**
+     * Gets the latest state of a long-running operation.  Clients can use this
+     * method to poll the operation result at intervals as recommended by the API
+     * service.
+     *
+     * Sample code:
+     * ```
+     * try {
+     *     $operationsClient = new OperationsClient();
+     *     $name = "";
+     *     $response = $operationsClient->getOperation($name);
+     * } finally {
+     *     if (isset($operationsClient)) {
+     *         $operationsClient->close();
+     *     }
+     * }
+     * ```
+     *
+     * @param string $name         The name of the operation resource.
+     * @param array  $optionalArgs {
+     *                             Optional.
+     *
+     *     @type \Google\GAX\RetrySettings $retrySettings
+     *          Retry settings to use for this call. If present, then
+     *          $timeoutMillis is ignored.
+     *     @type int $timeoutMillis
+     *          Timeout to use for this call. Only used if $retrySettings
+     *          is not set.
+     * }
+     *
+     * @return \google\longrunning\Operation
+     *
+     * @throws \Google\GAX\ApiException if the remote call fails
+     */
+    public function getOperation($name, $optionalArgs = [])
+    {
+        $request = new GetOperationRequest();
+        $request->setName($name);
+
+        $mergedSettings = $this->defaultCallSettings['getOperation']->merge(
+            new CallSettings($optionalArgs)
+        );
+        $callable = ApiCallable::createApiCall(
+            $this->operationsStub,
+            'GetOperation',
+            $mergedSettings,
+            $this->descriptors['getOperation']
+        );
+
+        return $callable(
+            $request,
+            [],
+            ['call_credentials_callback' => $this->createCredentialsCallback()]);
+    }
+
+    /**
+     * Lists operations that match the specified filter in the request. If the
+     * server doesn't support this method, it returns `UNIMPLEMENTED`.
+     *
+     * NOTE: the `name` binding below allows API services to override the binding
+     * to use different resource name schemes, such as `users/&#42;/operations`.
+     *
+     * Sample code:
+     * ```
+     * try {
+     *     $operationsClient = new OperationsClient();
+     *     $name = "";
+     *     $filter = "";
+     *     foreach ($operationsClient->listOperations($name, $filter) as $element) {
+     *         // doThingsWith(element);
+     *     }
+     * } finally {
+     *     if (isset($operationsClient)) {
+     *         $operationsClient->close();
+     *     }
+     * }
+     * ```
+     *
+     * @param string $name         The name of the operation collection.
+     * @param string $filter       The standard list filter.
```

Comment:
Hmmm looks to me like this should not be required, but it is marked as such in our gapic config (https://github.com/googleapis/googleapis/blame/master/google/longrunning/longrunning_gapic.yaml#L59). @garrettjonesgoogle what do you think?